device/isis/v4routerange.yaml

```diff
@@ -36,4 +36,11 @@ components:
         n_flag:
           x-include: ./routerange.yaml#/components/schemas/Isis.RouteRange/properties/n_flag
           x-field-uid: 9
+        prefix_sid_params:
```

Comment:
I think prefix_sids is good. 
What is the usecase for one prefix being associated with multiple sids  ?

Reply:
one prefix can be part of different paths associated by different algorithms (and sids).